### ISSUE TITLE
Suggestion: Add Feature Scaling for Logistic Regression

### ISSUE THREAD

**Issue by saivineethreddysuravi** [HarshaKoppula2/-Customer-Churn-Analysis-for-Subscription-Based-Services#1]
Yo Harsha, just checking out your churn analysis repo! The structure looks clean.

One small tip: I noticed you're passing numerical features directly into the Logistic Regression model. Since that model relies on gradients, adding a `StandardScaler` to your preprocessing pipeline usually helps it converge faster and can bump up your accuracy.

I tweaked that part of the code if you want to test it out:

```python
from sklearn.preprocessing import StandardScaler # Add this import

# ... (rest of imports)

# Update the preprocessor to scale numerical features
preprocessor = ColumnTransformer(
    transformers=[
        # Use StandardScaler for numerical columns instead of 'passthrough'
        ('num', StandardScaler(), numerical_features),
        ('cat', OneHotEncoder(handle_unknown='ignore'), categorical_features)
    ])
```

Hope this helps!
